.env

```
@@ -0,0 +1 @@
+VITE_API_URL=http://localhost:4000/api
```

Can you add the end of the line @mayura-andrew 
```suggestion
VITE_API_URL=http://localhost:4000/api

```